src/Run.purs

```diff
@@ -1,41 +1,43 @@
 module Run
   ( Run(..)
   , run
-  , runBase
-  , runBaseWithCont
-  , runWithBase
+  , runEffect
+  , runEffectRec
+  , runEffectCont
   , runWithEffect
-  , interpretWithBase
   , interpretWithEffect
   , foldWithEffect
-  , liftEffect
-  , liftBase
+  , lift
   , peel
   , resume
   , send
   , expand
-  , BaseEff
-  , BaseAff
+  , EFF
+  , AFF
+  , liftEff
+  , liftAff
+  , runBaseEff
+  , runBaseAff
+  , runBaseAff'
```

Comment:
When i saw this, I thought it was going to do something specific to `Aff a b`. Like require a canceler or some conversion `Run (aff :: AFF e) a -> Run (eff :: EFF (exception :: EXCEPTION | e) (Canceler e)` or something. I think what the function does is fine, but the name is a bit confusing. I don't have a better suggestion though.

Reply:
Maybe it should be `toEff` and `toAff`?